odoo/tools/pdf.py

```diff
@@ -161,9 +175,9 @@ def addAttachment(self, name, data, subtype=""):
                 NameObject("/AF"): attachment_array
             })
 
-    def embed_odoo_attachment(self, attachment):
+    def embed_odoo_attachment(self, attachment, subtype=None):
```

Comment:
As a general rule we tend to avoid doing such API changes in stable (as it could break third-party code that is extending/hooking it), but since this is not a model method, and it was added recently, I think it's acceptable in this case.